iotdb-core/confignode/src/main/java/org/apache/iotdb/confignode/manager/ConfigManager.java

```diff
@@ -248,6 +249,8 @@ public class ConfigManager implements IManager {
 
   private static final String DATABASE = "\tDatabase=";
 
+  private ProcedureMetrics procedureMetrics;
```

Comment:
why we put this into ConfigManager rather than ProcedureManager?

Reply:
Referring to NodeMetrics and PartitionMetrics. Put it in ConfigManager for easy access to addMetrics and removeMetrics

Reply:
fixed